obelisk/python/obelisk_py/core/node.py

```diff
@@ -71,6 +81,92 @@ def t(self) -> float:
         """Get the current time in seconds."""
         return self.get_clock().now().nanoseconds / 1e9
 
+    # ###################### #
+    # COMPONENT REGISTRATION #
+    # ###################### #
+
+    def register_obk_publisher(
+        self,
+        ros_parameter: str,
+        key: Optional[str] = None,
+        msg_type: Optional[Type] = None,
```

Comment:
c++ will require the message type here, and will not be able to load it in via the config string. So this is fine to keep in the python, but I just wanted to say that field in the yaml will be unused in the c++ side

Reply:
resolved.